packages/perseus-editor/src/widgets/__tests__/image-editor.test.tsx

```diff
@@ -0,0 +1,165 @@
+import {ApiOptions, Dependencies, Util} from "@khanacademy/perseus";
+import {render, screen} from "@testing-library/react";
+import {userEvent as userEventLib} from "@testing-library/user-event";
+import * as React from "react";
+
+import {testDependencies} from "../../../../../testing/test-dependencies";
+import ImageEditor from "../image-editor/image-editor";
+
+import type {UserEvent} from "@testing-library/user-event";
+
+const realKhanImageUrl =
+    "https://cdn.kastatic.org/ka-content-images/61831c1329dbc32036d7dd0d03e06e7e2c622718.jpg";
+
+const apiOptions = ApiOptions.defaults;
+
+describe("image editor", () => {
+    let userEvent: UserEvent;
+
+    beforeEach(() => {
+        userEvent = userEventLib.setup({
+            advanceTimers: jest.advanceTimersByTime,
+        });
+
+        jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
+            testDependencies,
+        );
+    });
+
+    test("should render empty image editor", () => {
```

Comment:
for consistency in our code base, let's use `it` here and the remaining test scenarios.

```suggestion
    it("should render empty image editor", () => {
```

Reply:
nooooooo 😭